### ISSUE TITLE
isAnagramXor function fails for some test cases

### ISSUE THREAD

**Issue by prasadpande1990** [lakshmc/intwalgos#1]
Hi Lakshmi,

I was looking for similar solution to check if 2 strings are anagram using XOR logic.

When I came across your solution I can see that your solution will fail for some test scenarios.

For example,

string1 = "aa"
string2 = "bb"

or
string1 = "xaabby"
string2 = "xccddy"

Just wanted to let you know. Please share if you find the solution for this.

Thanks

**Comment by lakshmc** [comment]
Good catch. Thanks for pointing out. Will update the code if I find the solution.
